store/BUILD.bazel

```diff
@@ -25,6 +25,7 @@ go_test(
     ],
     embed = [":store"],
     flaky = True,
+    shard_count = 23,
```

Comment:
If we add a new test in store, should we increase the `shard_count` here?

Reply:
Normally, you don't need to worry about this. Unless you write a flaky test here, we will increase it when this package's test is slow.